packages/LablePicture/app/page.tsx

```diff
@@ -0,0 +1,247 @@
+"use client";
+
+import AppContext from "@/context/appContext";
+import React, { useState } from "react";
+import Editor from "@monaco-editor/react";
+import DEVELOPMENT_CONFIG from "@/package/development.config";
+import uniqid from 'uniqid';
+import { LocaleTypes, InsertMultipleContentVals, ContentType, GenerateNewTextKeysArgs } from "@/funcs/courseModuleTypes";
+import LabelComponent from "@/package/NewLablePicture"; // Import your LabelComponent
```

Comment:
Can you take copy from latest page.tsx and not change this component? refer to https://github.com/iClasser/iclasser-react-lms-components/blob/develop/packages/MatchingPairs/app/page.tsx#L7